src/components/MenuItems.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { List, Image } from "semantic-ui-react";
+
+const MenuItems = ({ item }) => {
+  return (
+    <List horizontal>
```

Comment:
So now you are mapping Lists while you should map List.Items. I say take `<List horizontal>` out of this component so it wraps all List.Items, thus I don't think you need this one  `<div data-cy={`item-${item.id}`} key={item.id}>`.

Reply:
We have moved <List horizontal> out of the MenuItem Component, and removed key={item.id}, but we think we need data-cy={item-${item.id}} to be able to choose which item we run tests on.